src/helpers/coordinates.ts

```diff
@@ -43,6 +48,41 @@ function isCharADigit(char: string) {
   return char >= "0" && char <= "9";
 }
 
+// we limit the max column to 3 letters and max row to 7 digits for performance reasons
+export const MAX_COL = lettersToNumber("ZZZ");
+export const MAX_ROW = 9999998;
+
+export function consumeSpaces(chars: TokenizingChars) {
+  while (chars.current === " ") {
+    chars.advanceBy(1);
+  }
+}
+
+export function consumeLetters(chars: TokenizingChars) {
+  if (chars.current === "$") chars.advanceBy(1);
+  if (!chars.current || !isCharALetter(chars.current)) {
+    return -1;
+  }
+  let letterPart = 0;
```

Comment:
nitpick: letterPart sounds like it would be the substring with the letters. Maybe just `colIndex` ?`currentcol` ?